Examples/DicomSeriesFromArray/DicomSeriesFromArray.py

```diff
@@ -0,0 +1,116 @@
+new_img = sitk.GetImageFromArray(new_arr)
+new_img.SetSpacing([2.5,3.5,4.5])
+
+
+# Write the 3D image as a series
+# IMPORTANT: There are many DICOM tags that need to be updated when you modify an
+#            original image. This is a delicate opration and requires knowlege of
+#            the DICOM standard. This example only modifies some. For a more complete
+#            list of tags that need to be modified see:
+#                           http://gdcm.sourceforge.net/wiki/index.php/Writing_DICOM
+
+writer = sitk.ImageFileWriter()
+# Use the study/series/frame of reference information given in the meta-data
+# dictionary and not the automatically generated information from the file IO
+writer.KeepOriginalImageUIDOn()
+
+# Copy relevant tags from the original meta-data dictionary (private tags are also
+# accessible).
+tags_to_copy = ["0010|0010", # Patient Name
+                "0010|0020", # Patient ID
+                "0010|0030", # Patient Birth Date
+                "0020|000D", # Study Instance UID, for machine consumption
+                "0020|0010", # Study ID, for human consumption
+                "0008|0020", # Study Date
+                "0008|0030", # Study Time
+                "0008|0050", # Accession Number
+                "0008|0060"  # Modality
+]
+
+modification_time = time.strftime("%H%M%S")
+modification_date = time.strftime("%Y%m%d")
+
+# Copy some of the tags and add the relevant tags indicating the change.
+# For the series instance UID (0020|000e), each of the components is a number, cannot start
+# with zero, and separated by a '.' We create a unique series ID using the date and time.
+# tags of interest:
+direction = new_img.GetDirection()
+series_tag_values = [(k, new_img.GetMetaData(k)) for k in tags_to_copy if new_img.HasMetaDataKey(k)] + \
+                 [("0008|0031",modification_time), # Series Time
+                  ("0008|0021",modification_date), # Series Date
+                  ("0008|0008","DERIVED\\SECONDARY"), # Image Type
+                  ("0020|000e", "1.2.826.0.1.3680043.2.1125."+modification_date+".1"+modification_time), # Series Instance UID
+                  ("0020|0037", '\\'.join(map(str, (direction[0], direction[3], direction[6],# Image Orientation (Patient)
+                                                    direction[1],direction[4],direction[7])))),
+                  ("0008|103e", "Created-SimpleITK")] # Series Description
+
+for i in range(new_img.GetDepth()):
+    image_slice = new_img[:,:,i]
+    # Tags shared by the series.
+    for tag, value in series_tag_values:
+        image_slice.SetMetaData(tag, value)
+    # Slice specific tags.
+    image_slice.SetMetaData("0008|0012", time.strftime("%Y%m%d")) # Instance Creation Date
+    image_slice.SetMetaData("0008|0013", time.strftime("%H%M%S")) # Instance Creation Time
+    image_slice.SetMetaData("0008|0060", "CT")  # set the type to CT so the thickness is carried over
```

Comment:
Setting the type to CT preserves the slice location, slice thickness is a different quantity. In DICOM nomenclature slice spacing and thickness are two different quantities. (0018, 0050) is the thickness of a single slice, (0018, 0088) slice spacing and (0020, 0032) image position patient determine the 3D spacing between slices.